src/tickable.ts

```diff
@@ -8,8 +8,52 @@ import { Vex } from './vex';
 import { Element } from './element';
 import { Flow } from './tables';
 import { Fraction } from './fraction';
+import { TickContext } from './tickcontext';
+import { ModifierContext } from './modifiercontext';
+import { Tuplet } from './tuplet';
+import { Voice } from './voice';
+import { Modifier } from './modifier';
+
+export interface LeftRight {
+  left: number;
+  right: number;
+}
+export interface Space {
+  used: number;
+  mean: number;
+  deviation: number;
+}
+export interface FormatterMetrics {
+  duration: string;
+  freedom: LeftRight;
```

Comment:
Do we need a new type name `LeftRight` here? Can we type it inline?

Reply:
Not really, I change it.

Reply:
Now inline.